salt/output/json_out.py

```diff
@@ -54,7 +54,7 @@ def output(data):
     '''
     try:
         if 'output_indent' not in __opts__:
-            return json.dumps(data, default=repr, indent=4)
+            return json.dumps(data, default=repr, indent=4, encoding='latin1')
```

Comment:
The data is gathered on the minion, so, yes, it makes sense to know what's the encoding the minion is reporting because that might help to properly decode the data on the minion side which will stop the tracebacks on the master side.